src/lib/y2firewall/clients/installation_finish.rb

```diff
@@ -43,7 +45,7 @@ def initialize
       end
 
       def title
-        "Writing Firewall Configuration..."
+        _("Writing Firewall Configuration...")
```

Comment:
The translation was available in "network" textdomain, but in this client we are using the firewall textdomain so not sure if it is ok, to just move the translation

Reply:
I think it's OK, just ping Stanislav Brabec, AFAIK he has some scripts which can move translations between textdomains.